openslides/utils/utils.py

```diff
@@ -1,6 +1,23 @@
+import re
 import roman
 
 
+first_re = re.compile('(.)([A-Z][a-z]+)')
+second_re = re.compile('([a-z0-9])([A-Z])')
+
+
+def convert_camel_case(text):
```

Comment:
The name is unclear. When I read the code, I though a string was converted to CamelCase. For example a function
```python
covert_lower_case(my_string)
```
should make my_string lower case and not expect a lower_case_string.

Proposal: Rename the function to camel_to_snake_case.